### PR TITLE
Only proxy defined methods

### PR DESCRIPTION
**Pull request by sperand-io**

### CODE REVIEW

File: lib/index.js [hunk 1]
```diff
@@ -41,7 +41,6 @@ GA.prototype.initialize = function(){
 
 GA.prototype.track = proxy('track');
 GA.prototype.identify = proxy('identify');
```
Comment (f2prateek) [inline lib/index.js]: Unrelated to the PR, but should we remove the identify proxy? I don't see it in either classic or universal.

Comment (sperand-io) [inline lib/index.js]: good catch! yep, looks like it!

**Comment by calvinfo** [comment]
wahooo :)

**Comment by f2prateek** [comment]
Closes #14.

**Comment by gjohnson** [comment]
@sperand-io @ndhoule Merged this because I've added some new logging and this is exploding right now. Also, made some style tweaks. :-)

@ndhoule I closed the other one.

**Pull request merged**